src/py42/clients/settings/device_settings.py

```diff
@@ -264,6 +283,13 @@ def _build_regex_exclusions(self):
         }
         self.data[u"backupPaths"][u"excludeUser"] = user_exclude_dict
 
+    @property
+    def locked(self):
+        """Indicates whether the backup set as a whole is locked. If True, individual
+        settings for this backup set (except for Destination settings), cannot be modified.
+        """
+        return u"@locked" in self.data
```

Comment:
Is there never a '@locked': 'false'?

Would it be better to check for to make sure the value is actually `true` anyway?